e2e/utils_test.go

```diff
@@ -23,15 +23,17 @@ import (
 	"time"
 
 	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/clastix/capsule/api/v1alpha1"
 )
 
 const (
-	defaultTimeoutInterval = 10 * time.Second
+	defaultTimeoutInterval = 35 * time.Second
```

Comment:
Please, don't increase this but rather set an override for the Pod recreation one.

Reply:
fixed, moved it to different variable `podRecreationTimeoutInterval`